src/HeaderRow.tsx

```diff
@@ -50,7 +50,7 @@ export default function HeaderRow<R, K extends keyof R>({
   return (
     <div
       className="rdg-header-row"
-      style={{ width, height }}
+      style={{ width, height, lineHeight: `${height}px` }}
```

Comment:
Needs `pre-canary` catchup?

Reply:
No?

Reply:
Nvm, for some reason I thought we fixed the line height in a separate PR